src/test/java/com/lob/model/USVerificationTest.java

```diff
@@ -13,7 +16,7 @@
     public void testUsVerification() throws Exception {
         LobResponse<USVerification> response = new USVerification.RequestBuilder()
                 .setRecipient("Donald")
-                .setPrimaryLine("185 Berry St")
+                .setPrimaryLine("deliverable")

```

Comment:
Why did this change? Is this a test address that never changes?

Reply:
Since the value is in the verification itself, we cannot give it for free in test mode. We've recently changed the way we do deliverabilities in _test_ mode to allow customers to fully test all possible values that the API could return. In the latest API Version (2018-03-01), providing a primary line of `deliverable` will return a deliverable address example. You can find more info [here](https://lob.com/docs#us-verification-test-environment).